docs/src/components/HeaderNavBar.tsx

```diff
@@ -156,6 +156,7 @@ export default function HeaderNavBar() {
             anchorEl={productsMenuRef.current}
             transition
             placement="bottom-start"
+            style={{ zIndex: 1200 }}
```

Comment:
mistake from the previous PR.

Reply:
We might be able to poll a value from theme.zIndex?